tests/test_engines.py

```diff
@@ -38,33 +38,33 @@ def test_espeak_voices(driver_name):
         pytest.skip(f"Skipping eSpeak-specific test for {driver_name}.")
 
     engine = pyttsx3.init(driver_name)
-    print(list(pyttsx3._activeEngines))
-    print(engine)
-    assert str(engine) == "espeak", "Expected engine name to be espeak"
-    voice = engine.getProperty("voice")
-    if voice:  # eSpeak-NG Windows v1.52-dev returns None
-        assert (
-            voice == "English (Great Britain)"
-        ), f"Expected {engine} default voice to be 'English (Great Britain)'"
+    assert str(engine) == "espeak", "Expected engine name to be 'espeak'"
+
+    # Retrieve and print voices without modifying `voice` property
     voices = engine.getProperty("voices")
-    print(f"{engine} has {len(voices) = } voices.")
-    # Linux eSpeak-NG v1.50 has 109 voices,
-    # macOS eSpeak-NG v1.51 has 131 voices,
-    # Windows eSpeak-NG v1.52-dev has 221 voices.
-    assert len(voices) in {109, 131, 221}, f"Expected 109, 131, 221 voices in {engine}"
-    # print("\n".join(voice.id for voice in voices))
+    print(f"{engine} has {len(voices)} voices.")
+    assert len(voices) in {
+        109,
+        131,
+        221,
+    }, f"Expected 109, 131, or 221 voices in {engine}"
+
+    # Filter English voices
     english_voices = [voice for voice in voices if voice.id.startswith("English")]
-    # Linux eSpeak-NG v1.50 has 7 English voices,
-    # macOS eSpeak-NG v1.51 and Windows eSpeak-NG v1.52-dev have 8 English voices.
-    assert len(english_voices) in {7, 8}, "Expected 7 or 8 English voices in {engine}"
+    assert len(english_voices) in {7, 8}, "Expected 7 or 8 English voices"
+
+    # Queue a single utterance for each voice without calling `runAndWait()` in a loop
     names = []
     for _voice in english_voices:
         engine.setProperty("voice", _voice.id)
-        # English (America, New York City) --> America, New York City
```

Comment:
This is much better documentation of what the code does than ~ Extract hame for display.